release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -316,6 +316,11 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+[id="tuning-of-power-states"]
+==== Power saving configurations
+
+In {product-title} {product-version}, by enabling C-states and OS-controlled P-states, you can use different power saving configurations for critical and non-critical workloads. You can apply the configurations through the new `perPodPowerManagement` workload hint, and the `cpu-c-states.crio.io` and `cpu-freq-governor.crio.io` CRI-O annotations. For more information about the feature, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#node-tuning-operator-pod-power-saving-config_cnf-master[Power saving configurations].
```

Comment:
Should this be 'operating system-controlled'? Just checking because I didn't find any other references to 'OS-controlled'.

Reply:
I think it's better to use "OS-controlled", it's a well known abbreviation: https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations#general-guidelines

I'm also using this version in the main docs that has already been merged.